test/common.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+
+# Loop over all plugins and call a test script ($1) with the given arguments ($2)
+function loop_over_plugins() {
+  local script=${1:-}
+  local opts=${2:-}
+
+  local basedir=$(basedir)
+
+  # Environment variable which can be used my plugins
+  export TEST_INFRA_SCRIPTS="$basedir/test-infra/scripts"
+
+  for plugin in "${basedir}"/plugins/*; do
+    local test_script="$plugin/test/$script"
+    if [ -x "$test_script" ]; then
+      echo "## $plugin ###############################"
+      eval "$test_script $opts" || fail_test
+      echo "##########################################"
+    fi
```

Comment:
I wonder if it's able to provide some kind of default test/$script under root test folder, e.g. a default `presubmit-test.sh`. If a plugin wants to use the default script or the plugin doesn't have a test script, we can use the default test script to test.